core/src/main/java/io/questdb/cairo/wal/WalPurgeJob.java

```diff
@@ -408,7 +415,14 @@ protected boolean runSerially() {
         final long t = clock.getTicks();
         if (last + checkInterval < t) {
             last = t;
-            broadSweep();
+            // do nothing if the snapshot is running (holding the lock)
```

Comment:
nit: shall we log an info message here, so that the user (and us) know that the purge job is blocked?